.github/workflows/staging-deploy.yml

```
@@ -0,0 +1,65 @@
+name: Staging - Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Staging — Docker build and push
+    types:
+      - completed
+
+env:
+  AWS_ACCOUNT_ID: ${{ vars.STAGING_AWS_ACCOUNT_ID }}
+  AWS_REGION: ca-central-1
+  CLUSTER_NAME: Forms
+  SERVICE_NAME: form-api
+  REGISTRY: ${{ vars.STAGING_AWS_ACCOUNT_ID }}.dkr.ecr.ca-central-1.amazonaws.com/forms/api
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy-form-viewer-service:
```

Not a big deal but we could rename this job :)
```suggestion
  deploy-form-api-service:
```

---

Ah good catch - changed!